nodes/stampzilla-magicmirror/webserver.go

```diff
@@ -50,6 +50,21 @@ func initWebserver() {
 		m.HandleRequest(c.Writer, c.Request)
 	})
 
+	r.GET("/proxy", func(c *gin.Context) {
```

Comment:
I would probably have used a `httputil.ReverseProxy` with a custom `Director`. 

example: 
```
	proxy := &httputil.ReverseProxy{
		Director: func(r *http.Request) {
 // get query parm url and make it into a new Url and set r.URL with it :) 
			r.URL = u
		},
	}

r.GET("/proxy", gin.WrapH(proxy))
```

httputil.ReverseProxy is rfc compliant and also support websockets from go 1.12 and forward.

Reply:
Cooool! Will try it out :D

Reply:
Can't get it to work :( I think the issue is that the `httputil.ReverseProxy` forwards to many headers and I don't want that. I just want a clean new request and then forward the result to the client.